src/olympia/amo/management/base.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+
+class BaseDataCommand(BaseCommand):
```

Comment:
There is already a base command class in `olympia/amo/management/__init__.py` so you don't need a new file

Reply:
TIL. I'll update it.

Reply:
Fixed